client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -153,6 +129,37 @@ class BusinessDetails extends Component {
 			} );
 	}
 
+	async pushProfileUpdates( additions = {} ) {
```

Comment:
I was a little caught off guard by this name `pushProfileUpdates`, but I get that `updateProfileItems` would create conflicts.

Maybe `persistProfileItems`?